### ISSUE TITLE
Give soldering iron a tool a level of 1 fine welding quality 

### ISSUE THREAD

**Issue by Katanasoul91** [CleverRaven/Cataclysm-DDA#41234]
Following to the arc welder and makeshift arc welder, the soldering iron could be next in the welding tools category

# Describe the solution you'd like
Give the soldering iron a level of 1 fine welding quality 

# Describe alternatives you've considered
Also add 1 fine welding quality to the Firearm repair kit, and Gunsmith repair kit.

# Additional context
I did use a soldering iron and see my father using a welder, they are pretty much the same except the level of fine detail each uses.

**Comment by kevingranade** [comment]
Does fine welding 1 represent soldering? That's pretty much the only kind of welding that soldering irons can do if there's any higher melting temperature medium used or even a larger work piece I don't think that a soldering iron is up to it.

**Comment by ennob** [comment]
"Welding" implies melting the base metal where as soldering does not (only the solder melts). The is also a huge temperature difference between the two processes. I don't think they should be mixed in game.

**Comment by Katanasoul91** [comment]
The answear is a little more complex:

tl:dr: Welding is when you use a low melting metal to join two workpiece. Acetyne and plasma torches cut metals, therefore should be their "metal cutting quality".

Explanation:
When you weld you are melting a soldering agent so it can flow into the joint between two pieces, and when it cool down it turn back solid and secure the two pieces. 
When you use a welder you are making electricity flow from an electrode (think of it as a long and hard solder bar) and the piece you want to weld, the resulting closed circuit will make electricity flow and melt the tip of the electrode welding the two pieces together. A soldering iron, you only have a heated tip who melt the solder.

On the other hand, using something like a torch you are applying extreme levels of heat to a metal cutting it off by melting it.

There are things you could fuse together using the welding system or the torch, but usually, the welding option is used because it is safer and allow people to undo their job while keeping the bigger part remain intact.

**Comment by ennob** [comment]
[This link](https://me-mechanicalengineering.com/differences-between-welding-soldering-and-brazing/) summarizes it nicely.

**Comment by Katanasoul91** [comment]
You are right ennob, it should be soldering quality then.
